Makefile

```diff
@@ -1,40 +1,41 @@
-# You may not be able to build with a modern Xcode, since we still support 10.6.
-# So, if we have Xcode 4.6 around, we should use it.
+# For the building the driver for MacOS 10.11+, the preferred
+# XCode version is 7.3.1.
 #
-# You can do this by downloading Xcode 4.6.3 as a dmg from Apple, then
-# copying the contents into /Applications, then taking your ancient
-# MacOSX10.6.sdk that you keep kicking around, and copying it into
-# /Applications/Xcode-4.6.3.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/. 
-# Don't you love backwards compatibility?
-
-XCODEBUILD_ANCIENT ?= /Applications/Xcode-4.6.3.app/Contents/Developer/usr/bin/xcodebuild
-XCODEBUILD_MODERN ?= xcodebuild
-
-# Ok, no joy there?
-ifeq (,$(wildcard $(XCODEBUILD_ANCIENT)))
-	XCODEBUILD_ANCIENT = xcodebuild
+# You can do this by downloading Xcode 7.3.1 as a dmg from Apple, then
+# copying the contents into /Applications as Xcode-7.3.1.
+# If you also want to build from "release_pre_10_11", you may want to
```

Comment:
One thing in the back of my head is wondering if we can build both 10.11+ and 10.6-10.10 in the same tree.  That way we could still have only one `.pkg` file that "knows" what to install on what system.  If not, it's not the end of the world, but it would be kind of nice, to be sure.

Reply:
see next reply